### PR TITLE
For #5892 & #6380: Updates error page template and allows SSL bypassing

### PR DESCRIPTION
**Pull request by sblatz**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/assets/high_risk_error_style.css [hunk 1]
```diff
@@ -125,66 +99,12 @@ button.inProgress {
...
-  font-size: smaller;
-  border-radius: 0;
-}
-
 /* On large screen devices (hopefully a 7+ inch tablet, we already center content (see #errorPageContainer above).
    Apply tablet specific styles here */
 @media (min-width: 550px) {
   button {
-    min-width: 160px;
+    margin: var(--moz-vertical-spacing) auto var(--moz-vertical-spacing);
```
Comment (NotWoods) [inline app/src/main/assets/high_risk_error_style.css]: You can shorten `margin: top horizontal bottom` to `margin: vertical horizontal` :)
```suggestion
    margin: var(--moz-vertical-spacing) auto;
```

File: app/src/main/assets/low_and_medium_risk_error_style.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  width: 100%;
+  text-align: center;
+  border-radius: 5px;
+  border-color: #312a65;
+  padding: 1rem;
+  font-family: sharp-sans;
+  background-color: #312A65;
+  color: #FFFFFF;
+  background-image: none;
+  margin: var(--moz-vertical-spacing) 0 var(--moz-vertical-spacing);
```
Comment (NotWoods) [inline app/src/main/assets/low_and_medium_risk_error_style.css]: ditto

File: app/src/main/assets/low_and_medium_risk_error_style.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+p {
+  line-height: 20px;
+  margin: var(--moz-vertical-spacing) 0;
+}
+
+
+button {
+  /* Force buttons to display: inline-flex so we can vertically & horizontally center their text */
+  display: block;
```
Comment (NotWoods) [inline app/src/main/assets/low_and_medium_risk_error_style.css]: nit: comment doesn't match value

**Comment by sblatz** [comment]
@darkwing for testing, I'd recommend badssl.com

**Comment by darkwing** [comment]
This is really awesome @sblatz!  Looks and functions really well.  I'll clean up a few CSS / JavaScript things while you're gone.  I also figured out why we were seeing that weird tap/highlight behavior:

RemoteDebugHTML

Whichever code or service we're getting the `badCertTechnicalInfo` info from is sending us back descriptions within a `<label>`.  A bit weird but nothing to block this PR on.

I'll work on this during my flight home!

**Comment by sblatz** [comment]
Just a heads up that the `badCertTechnicalInfo` is going to be coming from AC... probably a good place to start is `ErrorPages.kt`. If you can't figure it out quickly it's something I can look at when I'm back (definitely a nit IMO)!

**Comment by darkwing** [comment]
Superceded by #8062; I had issues pushing to this PR so opened a new one so I can finish this before @sblatz gets back from PTO.

**Pull request closed without merging**